### ISSUE TITLE
Inquiry on downloading dataset

### ISSUE THREAD

**Issue by Kami-code** [ekosman/GraphVid#1]
Hi @ekosman . Thanks for your wonderful work on the graph video representation. I follow your instruction on downloading video dataset using [this repository](https://github.com/ekosman/KineticsDownloader). But some missing file error as shown below, can you help me deal with that? Appreciate.

![2023-10-18 13-51-22](https://github.com/ekosman/GraphVid/assets/58084462/7a33e714-17b5-4e09-9741-dd30b08726c8)

**Comment by ekosman** [comment]
True. this was a large file that I wasn't able to push to github freely. You can access the original repo with all resources here:

https://drive.google.com/drive/folders/1AMabmnJVejkZsfD0np1PrJSIfI5RrAQc?usp=sharing
